broadinstitute/pearl
# JN-364 adding demo shell portal

This adds a new portal to our seed files -- "demo".  It's currently a clone of ourhealth with stableIds changed, and some theme changes so that it's easily visually distinguishable.  The short term need for this is a portal that has multiple versions of the same survey so I can test some workflow changes (will add those in a later PR along with the admin fix).  Long term, this might evolve into a more sophisticated test harness, as well as a template for somewhat-neutral sandbox portal we could give access to potential customers to play around.  (that's why it's called 'demo' rather than test)

This does not yet include support for b2c login

TO TEST:
1. run `./scripts/populate_portal demo` 
2. go to `https://sandbox.demo.localhost:3001/`. 
3. confirm you see an ourhealth-like study but with a different color scheme
![image](https://github.com/broadinstitute/juniper/assets/2800795/a80403a7-4720-4afb-a4ce-56a9f1285dff)
